src/components/Icon/Icon.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import * as Chakra from '@chakra-ui/core';
+
+export enum IconSize {
+  s = 's',
+  m = 'm',
+  l = 'l',
+}
+
+export type IconProps = Chakra.IconProps & {
+  size?: IconSize | string;
+};
+
+const getSize = (size: string) => {
+  if (size === 's') return '0.75rem';
+  if (size === 'm') return '1rem';
+  if (size === 'l') return '1.25rem';
+  return size;
```

Comment:
otherwise return a default size 1em?

Reply:
There is a default size applied in props `size = IconSize.m`